source/administration/object-management/object-versioning.rst

```diff
@@ -266,6 +265,129 @@ enable versioning on only a prefix or subset of objects in a bucket.
 Objects created prior to enabling versioning have a 
 ``null`` :ref:`version ID <minio-bucket-versioning-id>`.
 
+
+Exclude a Prefix From Versioning
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can exclude certain :ref:`prefixes <minio-admin-concepts-organize-objects>` from versioning using the :ref:`MinIO Client <minio-client>`.
+This is useful for Spark/Hadoop workloads or others that initially create objects with temporary prefixes. 
+
+.. admonition:: Object locking
+   :class: note
+
+   Buckets with :ref:`object locking enabled <minio-object-locking>` require versioning and do not support excluding prefixes.
+
+- Use :mc-cmd:`mc version enable` with the :mc-cmd:`~mc version --excluded-prefixes` option:
+
+  .. code-block:: shell
+     :class: copyable
+
+     mc version enable --excluded-prefixes ALIAS/BUCKET "prefix1, prefix2"
```

Comment:
I think the syntax has to be:

```suggestion
mc version enable --excluded-prefixes "prefix1,prefix2" ALIAS/BUCKET
```

```
mc version enable --excluded-prefixes play/version-test "foo"                             1 ✘  docs   3.8.16   
mc: <ERROR> Unable to enable versioning: `SetVersion` is not supported for `filesystem`.
```

I _think_ we generally recommend doing all params prior to the ALIAS/BUCKET for parsing reasons, but I am not entirely sure if that's stlil a requirement.